### ISSUE TITLE
Unable to run Ansible

### ISSUE THREAD

**Issue by tonglil** [Shippable/support#628]
Hi, I want a minion to run an ansible playbook to deploy to a staging server after success, however when I try to run the `ansible-playbook` command, Shippable fails as if Ansible was not installed with:

![screen shot 2014-10-02 at 6 31 54 pm](https://cloud.githubusercontent.com/assets/3250776/4500913/180b16c6-4a9d-11e4-8ce0-cc71b8741f0e.png)

However, if I run the command under sudo, it will find Ansible, but it is unable to execute the playbook because of a credentials issue (that I believe is not the issue) because its running as sudo.

![screen shot 2014-10-02 at 6 36 46 pm](https://cloud.githubusercontent.com/assets/3250776/4500938/c3f112a6-4a9d-11e4-9a45-4f139526f6ec.png)

Basically, why does the minion think Ansible is not installed if I run it without sudo?
